### DISCUSSION TITLE
"social mask shells" Cheng Yufei&Wang Yizhang

### DISCUSSION THREAD

**Discussion by chefeifei-cyf** [realitydeslab/playshop-summer-2024#14] [Proposal]
Chinese Title: 辅助社交面部系统
Your Team:Cheng Yufei Wang Yizhang
Research Questions:
How to solve the social communication between people with social disabilities and normal people in the future?

Keywords:
Social isolation, social disabilities, social communication
Artefact / Media:
Film Essay, model,picture

Abstract
Introduction
Social research
Models of social isolation include both objective social contact and subjective perceived adequacy of contact. Zavaleta et al.defined social perceived adequacy of contact. Zavaleta et al.defined social isolation as “inadequate quality and quantity of social relations with other people at the individual, group,community,and larger social environment levels where human interaction takes place”
social isolation can be either an active or passive separation from society that limits social engagement, active participation,and interpersonal communication and can have detrimental effects on an individual’s physical or mental health.
Loneliness is associated with various health impacts, and it is important to highlight possible variables that place individuals at increased risk to adverse health outcomes. In addition to social isolation, there are a number of risk factors for loneliness. For example, living in a rural area, poor functional status (especially cognitive impairment), widowhood, being female, subjective causes (illness, death, etc.), depression, feeling misunderstood by others, and living alone (quarantine/social isolation for our purposes) . Indeed, a recent study reported similar risk factors associated with high rates of loneliness during the COVID-19 pandemic: being female, being younger, having fewer family resources (less contact with relatives), having fewer personal resources, and having a negative self-perception of aging. The main concern is that because of the current COVID-19 pandemic, loneliness is becoming an increasing and/or worsening problem ; this will have many acute and long term effects on individuals and may lead to other illnesses that have already been associated with loneliness (e.g., GAD) .
Future Prediction
With the development of virtual technology in the future, people can communicate normally in the virtual world, thereby achieving the purpose of socializing and doing activities with others without leaving home. More and more people are socializing in the virtual world. After escaping from reality, some people became overly anxious and overly nervous. They were unable to express their emotions normally through social interactions in the real society. Therefore, they began to avoid communication with the real society, causing society to begin to experience a kind of virtual reality. Social isolation caused by technology. This type of people are unable to have normal social emotional communication and self-emotional expression with normal people. At the same time, normal people are also unable to feel and understand the social expressions of this special group of people, resulting in greater social communication barriers. So how should designers reflect on and discuss this phenomenon? 
FACTORS
Facial Expression
“Social and emotional functions in facial expression and communication: The readout hypothesis” This paper introduces the importance of facial expressions in communication and discusses the differences between the classic emotional view and the behavioral ecological view. The classical emotion perspective considers facial expressions to be direct expressions of emotional states, whereas the behavioral ecological perspective considers facial expressions to be used voluntarily based on intention and context.A very meaningful hypothesis is put forward - Readout Hypothesis.
Facial expression as an external display of emotional state (Emotion II): Facial expression is an external readout of emotional state, showing specific emotions through the movement of facial muscles. These expressions are a product of evolution and aid in the communication and understanding of emotional states during social interactions.Spontaneous expression and symbolic expression: Spontaneous Expression: refers to the involuntary expression of emotional states and is a direct readout of the motivation-emotion system. For example, facial expressions when suddenly frightened. Symbolic Expression: Refers to voluntarily controlled expressions that reflect social intentions and situations. For example, smiling politely in certain social situations.
design process
<img width="383" alt="968cf160ca55d2d39ba29f6a27d80ee" src="https://github.com/realitydeslab/playshop-summer-2024/assets/173657105/163b298c-c59b-41a0-a90c-e578cc0de95b">

Inpiration
Proposed modification of facial social expression function for future socially impaired people in response to emotional expression disorders focusing on facial expressions.Since this phenomenon of speculation is a manifestation of the nature of the group, the design of speculation for the group should be more oriented towards the creation of a more universal system design or a unified device.
Unified and integrated design emphasises considering the problem as a whole, focusing on the relationships and interactions between the various components within the system. This avoids addressing a single problem in isolation and ignoring the impact of other related problems. By thinking systemically, design can better identify and address the root causes of problems, not just the surface symptoms.A unified and integrated design can better take into account and accommodate the needs and interests of different groups. Through a comprehensive participatory and consultative process, social conflicts arising from the neglect of the needs of certain groups can be reduced and social equity and inclusive development can be promoted.In the face of the complexity and variability of social problems, unified and integrated design has a strong capacity for dynamic adaptation. Through continuous monitoring and evaluation, timely adjustment and optimisation of design solutions, it can be more flexible in responding to changes and emerging issues, ensuring the long-term effectiveness of the solutions.
Concept Proposal
It is hypothesised that in the future there will be an increasing number of head-mounted devices for the socially challenged, similar to masks and hoods for facial expression-based socialisation, with the unifying function of addressing this social isolation, but it is hypothesised that there will be an unlimited number of new devices of this type in any form.
In response to this speculative design idea, we therefore propose a speculative design concept of “Assisted Social Shell”, i.e., an analogy between all forms of such aids to socialisation that encompass all such aids to socialisation with a unifying solving purpose, and the variety of externalised forms of a unifying device.
But since this is a systematic design, a speculative design with an infinite number of possibilities, for the possibilities we created two different axes of demand lines, each axis calibrated in two opposite directions, constituting a dimensional space with four separate zones, i.e., forming a design matrix, in which we speculated on four potential design scenarios in each of the four zones.
Our speculations can be seen in the design matrix below and in the attached image visuals and some models and hand. 

Explaination
The following is an explanation and elaboration of this design matrix. The matrix focuses on two axes of need for future socially impaired people, one for the willingness to socialise divided into two extremes of refusal and mandatory initiative, and the other for personal privacy divided into two extremes of full transparency 
This design matrix therefore divides the existing design  
![a4d5fbfcbf2a7672e91fd548e2ce2c4](https://github.com/realitydeslab/playshop-summer-2024/assets/173657105/10c9cf77-7068-44aa-85fb-a4a57ffc65ac)
and openness and private closure.This design matrix therefore divides the existing design area into four different quadrants, so we can have four different types of speculative design outputs within the four different quadrants
First of all in the upper left corner of the area depicted in an LCD screen completely closed helmet, it can only accept and record the information received by the user in the outside world, and can not respond to any, people are more like a machine to record the behaviour of others, and will be recorded information to return home to be processed and digested, which is designed to show a closed refused to socialise with the user’s needs.
In the upper right quadrant, a social replacement mask is presented that projects information in real time, projecting the words, expressions, and things that the user wants to share, i.e., replacing the medium of socialising with the mask.
In the lower left corner, in the social needs space, a technique based on neuromuscular electrical control is used to correctly translate the inner feelings of people with social disorders into facial expressions, which realise a more subjective will in an obsessive form.
Finally, in the lower right corner, in the corner of social needs, it is envisioned that some artistic mandatory structures to convey the emotions of people with social disorders for active or mandatory socialisation, it is more like an expression of emotions, so that more people can feel a catharsis of social will from this exaggerated form of their devices.

 REFERENCES 
Virtual and Augmented Reality in Social Skills Interventions for Individuals with Autism Spectrum Disorder: A Scoping Review Published: 16 November 2021  Volume 52, pages 4692–4707, (2022)
Miers, A.C., Sumter, S.R., Clark, D.M. et al. Interpretation Bias in Online and Offline Social Environments and Associations with Social Anxiety, Peer Victimization, and Avoidance Behavior. Cogn Ther Res 44, 820–833 (2020). https://doi.org/10.1007/s10608-020-10097-1
The State of the Art of Natural Language Processing—A Systematic Automated Review of NLP Literature Using NLP Techniquesan Sawicki,Maria Ganzha,Marcin Paprzycki
 Manipulating facial musculaturewith functional ewith functional electrical stimulation as an intervention for major depressive disorder:a focused search of literature for a proposal Ilya Demchenko1,2, Naaz Desai3,4, Stephanie N. Iwasa4,5, Fatemeh Gholamali Nezhad1, José Zariffa4,5,6,7,8, Sidney H. Kennedy1,2,9, Nicholas O. Rule10, Jeffrey F. Cohn11, Milos R. Popovic2,4,5,7,8, Benoit H. Mulsant9,12 and Venkat Bhat1,2,3,4,5,9Psy. 2012. Gangnam Style. Video. (15 July 2012.). Retrieved August 22, 2014 from https://www.youtube.com/watch?v=9bZkp7q19f0
Social and emotional functions in facial expression andcommunication:The readout hypothesisArticle in Biological Psychology·November 1994

**Comment by wuzhuo11** [comment]
impressive work!

**Comment by XiaoXiao-007** [comment]
For me who has social disabilities, I may not only have language problems, but also terrible eye contact.

**Reply by chefeifei-cyf** [reply]
The over-all-face mask may become a newer generation of designs

**Comment by wuzhuo11** [comment]
I'm a student, can you give me a free one?

**Comment by botaohu** [comment]
[Lucy https://inspirationist.net/hybrid-architectural-fashion-by-lucy-and-bart/](https://inspirationist.net/hybrid-architectural-fashion-by-lucy-and-bart/)

**Comment by chefeifei-cyf** [comment]
If one is to design a  unified but stylised device for the future of social interaction,maybe this masks or helmets can inspire you a lot.

**Comment by chefeifei-cyf** [comment]
![2722e9f8994fd4fcc13980adc8d1936](https://github.com/realitydeslab/playshop-summer-2024/assets/173657105/00a3e346-bbd1-49c6-aa5b-8a6bb7edfc51)
![229d3ef80288ff43ead1a1da45c695e](https://github.com/realitydeslab/playshop-summer-2024/assets/173657105/2583eb08-6eb1-4958-88ae-bc71048563df)

**Comment by chefeifei-cyf** [comment]
![d59919be31b03c9bdc758fcbb42ddd1](https://github.com/realitydeslab/playshop-summer-2024/assets/173657105/b0bc5661-c200-419f-911a-dd715a447d7c)

**Comment by chefeifei-cyf** [comment]
If one is to design a  unified but stylised device for the future of social interaction,maybe this masks or helmets can inspire you a lot.people in social disablities can choose their own social mask shells to protect themselves away from the embarrassing situation.

**Comment by glitch-kraft** [comment]
**Title**
Assisted Social Shell

**Tagline**
Enhancement of social skills through assisted electro-muscular facial stimulation via electronic liquid crystal enabled face gear to overcome the dialectics of introvercy-extrovercy

**Comment by chefeifei-cyf** [comment]
[social mask shells论文.pdf](https://github.com/user-attachments/files/16082640/social.mask.shells.pdf)
